.github/workflows/add-to-project.yaml

```
@@ -0,0 +1,25 @@
+name: Add issue to project
+
+on:
+  workflow_call:
+    inputs:
+      project_id:
+        type: number
+        description: The ID of the project to add the issue to
+        default: 1
```

`janus-idp planning` is https://github.com/orgs/janus-idp/projects/2

```suggestion
        default: 2
```

---

ah my bad